main.go

```diff
@@ -169,6 +170,14 @@ func main() {
 		mgrOptions.NewCache = cache.MultiNamespacedCacheBuilder(strings.Split(watchNamespace, ","))
 	}
 
+	leaseTime, found := os.LookupEnv("LEASE_TIME")
```

Comment:
To be consistent with the configuration mechanism that the operator already uses I would propose to change this to a flag.